### PR TITLE
[Task-9] The Navbar ui has been implemented

### PR DESCRIPTION
**Pull request by karthicravenantech**

### CODE REVIEW

File: front/src/components/CustomCard/CustomCard.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+
+interface CustomCardProps {
+  logo: string; // Explicitly typing the logo prop as a string
+  appName: string;
+}
+
+const CustomCard: React.FC<CustomCardProps> = ({ logo, appName, content }) => {
```
Comment (esakkirajaravenantech) [inline front/src/components/CustomCard/CustomCard.tsx:11]: Remove unused argument

File: front/src/components/CustomCard/CustomCard.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+
+interface CustomCardProps {
+  logo: string; // Explicitly typing the logo prop as a string
```
Comment (esakkirajaravenantech) [inline front/src/components/CustomCard/CustomCard.tsx:7]: Remove commented text

File: front/src/components/Header/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,168 @@
+import * as React from "react";
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import CssBaseline from "@mui/material/CssBaseline";
+import Drawer from "@mui/material/Drawer";
+import IconButton from "@mui/material/IconButton";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
```
Comment (esakkirajaravenantech) [inline front/src/components/Header/Header.tsx:8]: Remove unused imports

File: front/src/components/Header/Header.tsx [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+  const drawer = (
+    <Box sx={{ textAlign: "center" }}>
+      <Typography variant="h6" sx={{ my: 2 }}>
+        Authmika
+      </Typography>
+      <Divider />
+      <List>
+        {navItems.map((item) => (
+          <Link key={item.route} href={item.route}  sx={{ textDecoration: 'none', color: 'inherit', cursor: 'pointer' }}>
+            <ListItemButton component="a" disablePadding>
```
Comment (esakkirajaravenantech) [inline front/src/components/Header/Header.tsx:61]: Fix eslint error.

File: front/src/components/Header/Header.tsx [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+              boxSizing: "border-box",
+              width: drawerWidth,
+            },
+          }}
+        >
+          {drawer}
+        </Drawer>
+      </nav>
+    </Box>)
+  );
```
Comment (esakkirajaravenantech) [inline front/src/components/Header/Header.tsx:167]: Fix eslint error

File: front/src/components/CustomCard/CustomCard.tsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  appName: string;
+}
+
+const CustomCard: React.FC<CustomCardProps> = ({ logo, appName, content }) => {
+  return (
+    <Card variant="outlined" style={{ marginTop: '25px' }}>
+      <CardContent style={{ textAlign: 'center' }}>
+        <img src={logo} alt="Logo" style={{ width: '100%', height: '100px' }} />
+        <Typography variant="h5" component="div">
```
[the remark this answers is not in the document]
Comment (karthicravenantech) [reply front/src/components/CustomCard/CustomCard.tsx]: Changed next/image

File: front/src/components/CustomCard/CustomCard.tsx (hunk 1 above)
Comment (karthicravenantech) [reply front/src/components/CustomCard/CustomCard.tsx]: Removed unused argument

File: front/src/components/CustomCard/CustomCard.tsx (hunk 2 above)
Comment (karthicravenantech) [reply front/src/components/CustomCard/CustomCard.tsx]: Removed commented text

File: front/src/components/Header/Header.tsx (hunk 3 above)
Comment (karthicravenantech) [reply front/src/components/Header/Header.tsx]: Removed unused imports

File: front/src/components/Header/Header.tsx [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+    signOut({ redirect: true, callbackUrl: config.signoutCallback });
+  };
+  const [mobileOpen, setMobileOpen] = React.useState(false);
+  const [anchorEl, setAnchorEl] = React.useState(null);
+
+  const handleDrawerToggle = () => {
+    setMobileOpen(!mobileOpen);
+  };
+
+  const handleClick = (event) => {
```
[the remark this answers is not in the document]
Comment (karthicravenantech) [reply front/src/components/Header/Header.tsx]: Eslint error fixed

File: front/src/components/Header/Header.tsx (hunk 4 above)
Comment (karthicravenantech) [reply front/src/components/Header/Header.tsx]: Fixed eslint error

**Pull request merged**